### PR TITLE
fix svg overrides

### PR DESCRIPTION
**Pull request by mazmassa**
We are fixing some issues with svgs

### CODE REVIEW

File: src/components/Icons/Svg/Massa/MassaLogo.tsx [hunk 1]
```diff
@@ -1,6 +1,18 @@
-export function MassaLogo({ ...props }) {
+type SVGProps = {
+  size?: number;
+  className?: string;
+  rest?: any;
```
Comment (maxime-carabina) [inline src/components/Icons/Svg/Massa/MassaLogo.tsx:4]: You don't need to send the `rest` and `className`.
Normally, this is done automatically ?

File: src/components/Icons/Svg/Massa/MassaWallet.tsx [hunk 2]
```diff
@@ -1,6 +1,18 @@
...
   return (
-    <div className={`bg-primary w-fit rounded-full`} data-testid="svg-icon">
+    <div
+      className={`bg-primary w-fit rounded-full`}
+      data-testid="svg-icon"
+      {...rest}
+    >
       <svg
         width={props.size ?? 40}
         height={props.size ?? 40}
```
Comment (maxime-carabina) [inline src/components/Icons/Svg/Massa/MassaWallet.tsx:18]:
```suggestion
        width={size ?? 40}
        height={size ?? 40}
```

Comment (mazmassa) [reply src/components/Icons/Svg/Massa/MassaWallet.tsx:18]: Ohh! 

Fixed!

File: src/components/Icons/Svg/Massa/MassaLogo.tsx (hunk 1 above)
Comment (mazmassa) [reply src/components/Icons/Svg/Massa/MassaLogo.tsx]: Done!

**Review by maxime-carabina — APPROVED** [review]
Perfect 🎉 
LGTM !

**Pull request merged**